metals/src/main/scala/scala/meta/internal/metals/MetalsLanguageServer.scala

```diff
@@ -1313,6 +1313,15 @@ class MetalsLanguageServer(
   ): CompletableFuture[Unit] = {
     val path = AbsolutePath(event.path)
     val isScalaOrJava = path.isScalaOrJava
+    event.eventType match {
+      case EventType.Create =>
+        scribe.info(s"Created file: ${event.path}")
```

Comment:
Could we have it under `debug` ? Or is it only for testing now?

Reply:
I was thinking about leaving it as it is. It's not that much I think and in case of any problems users don't have to enable any flags

Reply:
I would change it to something like `[File watcher] Detected file creation` to be more exact. Just printing that the file was created is not that useful for the user.